Cargo.lock

```diff
@@ -15,9 +15,9 @@ dependencies = [
 
 [[package]]
 name = "ahash"
-version = "0.8.0"
+version = "0.8.3"
```

Comment:
Is this MSRV compatible?

Reply:
Oh, good point - I'll just revert the `cargo update`.  It's safest not to do it in a patch release anyway, I suppose, I've just apparently conditioned myself to do it when updating version numbers.  Done in e5ef2c9ba.